### DISCUSSION TITLE
Geodetic frame for Elevation/Azimuth calculations

### DISCUSSION THREAD

**Discussion by gwbres** [nyx-space/anise#308] [Q&A] [answered]
Could you please remind me what `Frame` I should use as a back up, to be able to resolve Elevation/Azimuth/Slant at all times ? (ie., backup geodetic frame).

My current implementation is to prefer `ITRF93` if we can fully define and retrieve it (requires internet access once a day). 
And I hesite between `IAU_EARTH_FRAME` and `EARTH_J2000` as the other "never failling / less precise" option

**Comment by ChristopherRabotin** [comment, marked as answer]
Of course! You should use the IAU EARTH FRAME for the long term and lower precision frame for AzElRange.

If you want a few months of predictions of the high fidelity frame, you can also download the EOP prediction : https://naif.jpl.nasa.gov/pub/naif/generic_kernels/pck/earth_200101_990827_predict.bpc. I forgot that file existed until recently, but we started using it. You'll find the details on this dataset here: https://naif.jpl.nasa.gov/pub/naif/generic_kernels/pck/earth_200101_990827_predict.cmt.
